traceapp/tmpl/doc.go

```diff
@@ -3,4 +3,4 @@
 // This package needs to be go generated after making changes to template files.
 package tmpl
 
-//go:generate go run -tags=dev data_generate.go
+//go:generate vfsgen -tags=dev -source="sourcegraph.com/sourcegraph/appdash/traceapp/tmpl".Data -output-tags=!dev
```

Comment:
I want to make sure I understand the CLI 100% here.

1. `-tags=dev` is needed because `traceapp/tmpl/data.go` has `// +build dev`?
2. `-source=...` specifies where to find the input VFS.
3. `-output-tags=!dev` says to write out a file that only builds with `// +build !dev`.

Is that correct?

One concern I have is that it's really hard to read `-tags=dev` and `-output-tags=dev` in the same line -- it is very hard to understand it as a newcomer and makes the whole thing look a bit magical. I wonder if (not now, but in the future), we could analyze the package for `tmpl.Data` and determine which build tags are needed automatically?